plugins/UM3NetworkPrinting/src/Cloud/CloudApiClient.py

```diff
@@ -71,21 +71,39 @@ def getClusters(self, on_finished: Callable[[List[CloudClusterResponse]], Any],
         :param on_finished: The function to be called after the result is parsed.
         """
 
-        url = "{}/clusters?status=active".format(self.CLUSTER_API_ROOT)
+        url = f"{self.CLUSTER_API_ROOT}/clusters?status=active"
         self._http.get(url,
                        scope = self._scope,
                        callback = self._parseCallback(on_finished, CloudClusterResponse, failed),
                        error_callback = failed,
                        timeout = self.DEFAULT_REQUEST_TIMEOUT)
 
+    def getClustersByMachineType(self, machine_type, on_finished: Callable[[List[CloudClusterWithConfigResponse]], Any], failed: Callable) -> None:
+        # HACK: There is something weird going on with the API, as it reports printer types in formats like
```

Comment:
There's a `product_to_id.json` in the XmlMaterialProfile plugin ;-p

Reply:
Yeah, the downside of that is that it doesn't work for all machines (so only for machines that are explicitly in that list)